migrations/7_deploy_creditor_proxy.js

```diff
@@ -0,0 +1,46 @@
+const CONSTANTS = require('./migration_constants');
+module.exports = (deployer, network, accounts) => {
+    const CreditorProxy = artifacts.require("CreditorProxy");
+    const ContractRegistry = artifacts.require("ContractRegistry");
+    const TokenTransferProxy = artifacts.require("TokenTransferProxy");
+    const DharmaMultiSigWallet = artifacts.require("DharmaMultiSigWallet");
+
+    const TX_DEFAULTS = { from: accounts[0], gas: 4000000 };
+
+    return deployer.then(async () => {
+        const registry = await ContractRegistry.deployed();
+        const proxy = await TokenTransferProxy.deployed();
+        const wallet = await DharmaMultiSigWallet.deployed();
+
+        await deployer.deploy(CreditorProxy, registry.address);
+        const creditorProxy = await CreditorProxy.deployed();
+
+        // Authorize creditor proxy to make `transferFrom` calls on the token transfer proxy.
+        let data = web3.sha3('addAuthorizedTransferAgent(address)').substring(0,10);
+        data += web3.padLeft(creditorProxy.address.substring(2), 64);
+
+        let receipt = await wallet.submitTransaction(proxy.address, 0, data);
```

Comment:
It doesn't look like you've transferred the ownership of `CreditorProxy` to the multisig wallet at this stage (you do that on line 44), and so you should just be able to call this method directly using `accounts[0]` I believe -- and then you can skip the timelock.

Reply:
Unfortunately, we're sending this tx to the `tokenTransferProxy` which *did* have it's ownership transferred in migration 6.

It's a little confusing since we have a variable just called `proxy`.. I updated this so the creditor proxy and token transfer proxy aren't so easily mistaken for each other.